admin_ui/src/components/MessagePopup.vue

```diff
@@ -1,7 +1,22 @@
 <template>
     <div
-        id="message_popup"
-        v-if="visible"
+        id="message_error"
+        v-if="visible && message.includes('error')"
```

Comment:
Rather than checking if the message contains 'error', we should use the `apiResponseMessage`, which has a `type` attribute. This will be `error` if there's an error.